docs/upgrading.md

```diff
@@ -6,6 +6,15 @@ the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summar
 
 ## Upgrading to v3.3
 
+### feat!: Remove deprecated config flags
+
+This PR removes the following options -
+
+- executorImage (use --executor-image flag to workflow-controller instead)
```

Comment:
could we include before and after examples?

Reply:
Yeah sure, I'll try my best. BTW one of the checks failed because I didn't format the fields of a structs properly. Now should I rebase the commit where I made changes related to that or make a new commit?

Reply:
both are fine